lms/djangoapps/grades/rest_api/v1/gradebook_views.py

```diff
@@ -564,15 +565,41 @@ def get(self, request, course_key):
                     grade_conditions['effective_grade__lte'] = request.GET.get('assignment_grade_max')
                 if request.GET.get('assignment_grade_min'):
                     grade_conditions['effective_grade__gte'] = request.GET.get('assignment_grade_min')
-                annotations['selected_assignment_grade_in_range'] = Exists(
-                    subqueryset.filter(
+            if request.GET.get('course_grade_min') or request.GET.get('course_grade_max'):
+                grade_conditions = {}
+                if request.GET.get('course_grade_min'):
+                    course_grade_min = float(request.GET.get('course_grade_min')) / 100
```

Comment:
Oh, wow, so these are stored in the DB as a ratio? I.e. the endpoint is translating the range of 0-50-100 -> 0-0.5-1? That's very surprising to me that it'd be stored that way. Weird!

Reply:
Yeah, the percent grade is a `FloatField` on the `PersistentCourseGrade` model. Out of curiosity, what about it surprises you? Is it because the subsection grades are stored as absolute numbers? Or is it something else?

Reply:
I don't necessarily always know what underlies my intuition. It's just surprising to me; not actionable feedback.